__tests__/Views/configuration/AdministrativeComponent-test.js

```diff
@@ -118,4 +120,30 @@ describe('test Administrative component', () => {
     wrapper.instance().sendDialToChatService('test', () => { });
     expect(mocksendDialfalse.mock.calls.length).toBe(1);
   });
+
+
+  test('check componentDidMount', async () => {
+    const instance = wrapper.instance();
+    await instance.componentDidMount();
+    expect(instance.state.upnp).toBe(true);
+  });
+
+
+  test('activate upnp', () => {
```

Comment:
it's not very important in this point but the correct spelling is UPnP, which stands for **U**niversal **P**lug a**n**d **P**lay 
```suggestion
  test('activate UPnP', () => {
```